### DISCUSSION TITLE
Switched "master" to "main" branch

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-distro#1510] [openHAB Maintainers Discussions]
Dear @openhab/maintainers,

We have just performed the renaming from `master` to `main` for all our repos (according to the guidelines put forward at https://github.com/github/renaming).
There should be automatic redirects in place, so that this step hopefully didn't break anything.
You might want to update the configuration of your local fork, though and scan through the code+documentation if there might be the need to update any urls to the new branch name.

Best regards,
Kai

**Comment by cweitkamp** [comment]
A quick guide to rename your local copy - assuming you are using `upstream` to keep your forks in sync:

```
git branch -m master main
git fetch upstream
git branch -u upstream/main main
```

**Comment by Confectrician** [comment]
If used somewhre,  you should also check the existing github actions.
They are probably still configured to run on push to the `master` branch.

**Comment by mueller-ma** [comment]
The Crowdin integration for openhab-android is still on master. It was configured by @openhab-bot, so I'm not allowed to change it.

**Comment by kaikreuzer** [comment]
Thanks for the hint! I have changed the crowdin configuration, you should have seen a million notifications from it... I hope it did all the right things, on openhab-core, it seems to be a bit confused and added many empty files 😕 .

**Comment by mueller-ma** [comment]
Crowdin uploaded the files again and now none of them are approved. It's probably better so switch back to master in the Crowdin. GitHub will redirect master -> main automatically.

**Comment by mueller-ma** [comment]
@kaikreuzer Can you remove the GitHub integration in Crowdin for openhab-android, so I can make some tests on my own?

**Comment by kaikreuzer** [comment]
@mueller-ma Ok, I have removed the integration on the android repo now. I can also activate it for the master branch again, if you wish, just let me know.

@ghys Did it also loose the approved translations for the webui repo? Let me know, if I need to do anything on that as well.

**Comment by mueller-ma** [comment]
I still cannot add a new connection. Can you enable it for `master` again and remove the files from `main`?

**Comment by kaikreuzer** [comment]
Ok - you are back on master.

**Comment by mueller-ma** [comment]
GitHub says that the redirect `master` to `main` if `master` is deleted, so I did that. Then Crowdin removed the integration. I recreated it on `main` and approved all current strings. It seems that a few percent of strings in every language are "lost", but can be restored by the translators from Crowdin TM.
It should be fine now.
